File: challenges/Python/Medium/العثور على العنصر الفريد في المصفوفة.py
def unique(arr: list[int]) -> list[int]:
    numbers = {}
    
    for i in arr:
        if i in numbers:
            numbers[i] += 1
        else:
            numbers[i] = 1
            
    result = []
    for key, value in numbers.items():
        result = [i for i in numbers if numbers[i] == value]
        
    return result

File: challenges/JavaScript/Medium/العثور على العنصر الفريد في المصفوفة.js
/*

Creator: OmarAfet
https://profile.satr.codes/OmarAfet/public/overview
https://github.com/OmarAfet

*/

function unique(arr) {
	let arr2 = [];
	for (let i = 0; i < arr.length; i++) {
		let count = 0;
		for (let j = 0; j < arr.length; j++) {
			if (arr[i] == arr[j]) {
				count++;
			}
		}
		if (count == 1) {
			arr2.push(arr[i]);
		}
	}
	return arr2;
}
